ftw/tests.sh

```diff
@@ -32,4 +32,4 @@ echo -e "\n[Ok] Got status code $status_code, expected 200. Ready to start."
 
 FTW_CLOUDMODE=${FTW_CLOUDMODE:-false}
 
-go-ftw run -d coreruleset/tests/regression/tests --config ftw.yml --read-timeout=10s --cloud=$FTW_CLOUDMODE || (echo "Envoy Logs:" && cat /home/envoy/logs/envoy.log; exit 1)
+go-ftw run -d coreruleset/tests/regression/tests --config ftw.yml --read-timeout=10s --cloud=$FTW_CLOUDMODE || (cp /home/envoy/logs/envoy.log /build/ftw-envoy.log; exit 1)
```

Comment:
Now that FTW actually runs to completion, catting the entire envoy logs on failure is too much so switched to copying it